ci-bench/src/util.rs

```diff
@@ -35,8 +35,358 @@ impl KeyType {
     }
 }
 
+pub mod async_io {
+    //! Async IO building blocks required for sharing code between the instruction count and
+    //! wall-time benchmarks
+
+    use async_trait::async_trait;
```

Comment:
same nit here w.r.t putting the `async_trait` import below the `std` imports with some separation.